presto-main/src/main/java/io/prestosql/connector/ConnectorManager.java

```diff
@@ -165,14 +165,14 @@ public synchronized void addConnectorFactory(ConnectorFactory connectorFactory)
         checkState(!stopped.get(), "ConnectorManager is stopped");
```

Comment:
can you please sort your commits with https://gist.github.com/findepi/7abf17c8a26d3b74e8bb9527caadfdbe

Otherwise it is hard to review this pull request when reviewing commit by commit.